met-web/src/components/engagement/form/EngagementFormTabs/AdditionalDetails/ConsentMessage.tsx

```diff
@@ -20,7 +20,7 @@ const ConsentMessage = () => {
                 <Box display="flex" flexDirection="column" justifyContent="space-between">
                     <RichTextEditor
                         handleEditorStateChange={handleRichContentChange}
-                        initialRawEditorState={consentMessage || ''}
+                        initialRawEditorState={initialConsentMessage || ''}
```

Comment:
Why is this called `initialConsentMessage`? There shouldn't currently be a default consent message or anything

Reply:
The `initialConsentMessage` variable is designed to hold either an empty string for new entries or the existing value when editing. The `consentMessage` variable, on the other hand, manages the current state of updates. When I attempted to use `consentMessage` for both initial and updated states, the RichTextEditor exhibited unusual behavior, notably resetting the cursor to the beginning during typing. This issue likely arises because the RichTextEditor re-renders every time the `consentMessage` state updates. To address this, I introduced the `initialConsentMessage` variable.

https://github.com/bcgov/met-public/assets/108045773/4408dfa1-a429-40cc-ae5d-b460b1d70df5